samples/subsys/power/stm32_pm/src/sample.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2018 Intel Corporation.
+ * Copyright (c) 2019 STMicroelectronics.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+
+#include <string.h>
```

Comment:
Nothing in this file is referenced except from `main.c`.  IMO its contents should be put into `main.c`.

Reply:
ok, removed the sample.h